src/Dotenv.php

```diff
@@ -40,22 +40,41 @@ public function __construct($path, $file = '.env')
 
     /**
      * Load environment file in given directory.
+     * If .env file is missing, skip loading and return false.
      *
-     * @return array
+     * @return boolean|array
      */
     public function load()
     {
-        return $this->loadData();
+        if (file_exists($this->filePath)) {
```

Comment:
I'm sorry, but I fail to see the race condition.

From [php doc](http://php.net/manual/en/function.file-exists.php):
```
Returns TRUE if the file or directory specified by filename exists; FALSE otherwise.
```

What is the possible exception? Do you think would be better to use [`is_readable`](http://php.net/manual/en/function.is-readable.php)?